examples/CRISP/client/package.json

```diff
@@ -12,15 +12,14 @@
     "cli": "pnpm sh ./scripts/cli.sh",
     "dev": "vite --no-open --host",
     "dev-static": "NO_HOT=1 vite --no-open --host",
-    "build:sdk": "pnpm -C ../packages/crisp-sdk build",
-    "build": "pnpm build:sdk && tsc && vite build",
+    "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "predeploy": "pnpm run build",
     "deploy": "gh-pages -d dist"
   },
   "dependencies": {
-    "@crisp-e3/sdk": "workspace:*",
+    "@crisp-e3/sdk": "0.2.0-test",
```

Comment:
Is doing this antithetical to having a monorepo? Does this not make testing much harder?

Reply:
I kept the same configuration but I see your point. Locally the client should use the local package as dependency, and ideally it should use the static version when deployed. We could open another issue for this maybe?